web/frontend/src/components/slam.vue

```diff
@@ -106,6 +116,77 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
   });
 };
 
+const executeMoveOnMap = async () => {
+  if (updatedDest === false) {
+    toast.error('please select a destination to move to');
+    return;
+  }
+  // get base resources
+  const baseResources = filterResources(props.resources, 'rdk', 'component', 'base');
+  if (baseResources === undefined) {
+    toast.error('No base component detected.');
+    return;
+  }
+
+  moveClick = !moveClick;
+
+  /*
+   * set request name
+   * here we set the name of the motion service the user is using
+   */
+  motionServiceReq.setName('builtin');
+
+  // set pose in frame
+  const destination = new commonApi.Pose();
+  const value = await fetchSLAMPose(props.name);
+  destination.setX(destinationMarker.x);
+  destination.setY(destinationMarker.y);
+  destination.setZ(destinationMarker.z);
+  destination.setOX(value.getOX());
```

Comment:
[q] do we need to set the orientation for destination

Reply:
it is not strictly necessary to set theta.. in this iteration of work on the SLAM UI for 2D motion planning.
But, I think it makes sense to still include it in the outgoing call, since it is the backend which chooses to exclude that information when executing motion planning.